nemo/collections/nlp/models/language_modeling/megatron_base_prompt_learning_model.py

```diff
@@ -211,68 +170,25 @@ def init_prompt_encoder(self):
         new_task = self.new_tasks[0]
         total_virtual_tokens = self.task_templates[new_task]["total_virtual_tokens"]
 
-        encoder_type = PromptEncoderType(self.cfg.p_tuning.get("encoder_type", "mlp").lower())
+        encoder_type = PromptEncoderType(self.cfg.p_tuning.get("encoder_type", "tpmlp").lower())
```

Comment:
Is there a reason why default was changed?

Reply:
No reason, it was during my copy based from the child model.